app/views/RoomMembersView/index.tsx

```diff
@@ -93,7 +93,7 @@ const RoomMembersView = (): React.ReactElement => {
 			members: [],
 			room: params.room || ({} as TSubscriptionModel),
 			end: false,
-			roomRoles: null,
+			roomRoles: undefined,
```

Comment:
```suggestion
			roomRoles,
```

Reply:
No value exists in scope for the shorthand property 'roomRoles'. Either declare one or provide an initializer.